scripts/download_test_resources.py

```diff
@@ -101,13 +107,40 @@ def download_test_resources(args: Args):
         sys.exit(1)
 
 
+def download_bioformats_resources() -> None:
+    from urllib.request import urlretrieve
+    from concurrent.futures import ThreadPoolExecutor
+
+    def _fetch(suffix: str) -> None:
+        ROOT = "https://downloads.openmicroscopy.org/images/"
+        dest = str(RESOURCES_DIR / suffix.replace("/", "_").replace("%20", "_"))
+        log.info(str(ROOT + suffix))
+        urlretrieve(str(ROOT + suffix), dest)
+
+    files = [
+        "ND2/aryeh/but3_cont200-1.nd2",
+        "ND2/jonas/header_test2.nd2",
+        "ND2/maxime/BF007.nd2",
+        "DV/siRNAi-HeLa/IN_02.r3d_D3D.dv",
+        "DV/siRNAi-HeLa/IN_02.r3d",
+        "Olympus-OIR/etienne/amy%20slice%20z%20stack_0001.oir",
+        "Imaris-IMS/davemason/Convallaria_3C_1T_confocal.ims",
+        "KLB/samples/img.klb",
+        "DICOM/samples/MR-MONO2-8-16x-heart.dcm",
```

Comment:
I will add these all to the package today. Mind if I push a commit to update the TEST_RESOURCES_HASH after I do? / If I can't push to this PR branch then I will just comment with the new hash.

Reply:
thanks! 
> Mind if I push a commit to update the TEST_RESOURCES_HASH after I do

of course not.  go for it.

Reply:
Don't think I have push access, new test resources hash: `9008c2bff8a87f7fecf6f1b111b6d8c1f8f7b49756939a84efc5e57ad878f017`

Reply:
thanks, added and reverted changes to download script